test/1.4/methods/T_idempotent.dml

```diff
@@ -0,0 +1,257 @@
+/*
+  © 2022 Intel Corporation
+  SPDX-License-Identifier: MPL-2.0
+*/
+dml 1.4;
+device test;
+import "testing.dml";
+
+header %{
+    static bool fired = false;
+%}
+extern bool fired;
+
+memoized method idemp_0() {
```

Comment:
`s/idemp/memo/` in this file?

Reply:
yup

Reply:
also rename the file from idempotent to memoized